django_components/component.py

```diff
@@ -55,15 +64,16 @@ def render_js_dependencies(self):
     def slots_in_template(template):
         return {node.name: node.nodelist for node in template.template.nodelist if is_slot_node(node)}
 
-    def compile_instance_template(self, slots_for_instance):
+    @lru_cache(maxsize=128)
```

Comment:
Is there a rationale for setting this to 128? Should be the number of unique times all your components get called with different values for template_name right? 

Maybe we could make this configurable via the COMPONENTS setting that @danjac added?

Reply:
It's the default value, but I think it does make sense to make it configurable.  I would bet that it won't matter for 95% of users, because they'll have < 10 templates ever used, but people who care at all will probably care a lot.

Reply:
Fully agree. So we add a default to COMPONENTS and always read from there?